check-matrix.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [[ -n ${GITHUB_ACTIONS} ]]; then
+    repo=${GITHUB_WORKSPACE}
+else
+    repo=$(dirname "$(readlink -f "${0}")")
```

Comment:
what's wrong with good ole `pwd`?

Reply:
I write all of my `bash` scripts under the assumption that they should work from wherever they are being called (i.e. `../ci/check-matrix.sh` does not work). I can change this to `${PWD}` if you prefer, given that it is unlikely that the script will be called from somewhere other than the repo.

Reply:
cool, no worries